src/mito2/src/worker.rs

```diff
@@ -563,6 +567,30 @@ impl<S: LogStore> RegionWorkerLoop<S> {
             BackgroundNotify::CompactionFailed(req) => self.handle_compaction_failure(req).await,
         }
     }
+
+    /// Handles internal command.
+    async fn handle_inter_command(&mut self, region_id: RegionId, command: Command) {
+        match command {
+            Command::SetReadonlyGracefully(req) => {
+                if let Some(region) = self.regions.get_region(region_id) {
+                    if region.is_writable() {
+                        region.set_writable(false);
+                    }
+
+                    let last_entry_id = region.version_control.current().last_entry_id;
+                    let _ = req.sender.send(SetReadonlyResult {
+                        last_entry_id: Some(last_entry_id),
```

Comment:
I realized that we actually don't need to return `last_entry_id`. A region always replays to the last entry id while opening as the WAL knows what's the current last entry id.

The [RFC](https://github.com/GreptimeTeam/greptimedb/pull/2703) didn't consider this point.

Reply:
Yes, but we need it while the new leader replays to verify whether it equals the actual.